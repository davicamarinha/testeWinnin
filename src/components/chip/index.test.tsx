import { render } from "@testing-library/react";
import Chip from "./";

describe("Chip Tests", () => {
  it("renders a snapshot", () => {
    const { container } = render(<Chip label="teste" />);
    expect(container).toMatchSnapshot();
  });
});
