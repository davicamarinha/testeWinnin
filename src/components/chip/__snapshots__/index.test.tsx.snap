// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chip Tests renders a snapshot 1`] = `
<div>
  <div
    class="text-white text-xs bg-primaryLight rounded-[4px] px-[8px] py-[3px] mr-2 mb-2"
    role="tag"
  >
    teste
  </div>
</div>
`;
